src/main/java/com/dev/cinema/model/Movie.java

```diff
@@ -0,0 +1,46 @@
+package com.dev.cinema.model;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "movie")
```

Comment:
якщо я правильно запам'ятав, коректно давати назви таблицям у множині
```suggestion
@Table(name = "movies")
```

Reply:
Agree 👍